ui-components/src/locales/ar.json

```diff
@@ -1123,7 +1123,8 @@
         "name": "المنطقة المحلية"
     },
     "progressNav": {
-        "srHeading": "تقدم"
+        "srHeading": "تقدم",
+        "current": "Current step: "
```

Comment:
Probably just filled the wrong one in here?

Reply:
yep